Robot_Adapter/CRUD/Create/Loads/Loadcase.cs

```diff
@@ -48,6 +48,14 @@ private bool CreateCollection(IEnumerable<Loadcase> loadCase)
                                 
                 int subNature;
                 IRobotCaseNature rNature = Convert.ToRobotLoadcaseNature(caseList[i], out subNature);
+
+                // Check if any loadcases havea zero
+                if(caseList.Any(lc => lc.Number == 0 || lc.Number < 1))
```

Comment:
This would prevent loadcases from 0.1-0.99 inclusive, is that the desired effect given the error message only mentions zero or negative numbers?

Reply:
The `Loadcase.Number` is an `int` - so intended.